### DISCUSSION TITLE
una idea para poder guardar los progresos

### DISCUSSION THREAD

**Discussion by deoke1983** [albertobsd/keyhunt#260] [Ideas]
hola Alberto,gran trabajo el tuyo!!!

no se programar,pero mi idea es para "autoguardar" los progresos,como en los videojuegos,el autoguardado,no es necesario que el programa cree un archivo con millones de claves....con guardar 1 sola clave cada.....5......10.....15 minutos? es mas que suficiente para no perder tooodo el trabajo realizado.
podria funcionar en todos los modos....excepto en random,yo lo que hago es,de vez en cuando,copio el rango en un txt y si lo paro porque tengo que usar el pc o reiniciarlo,al volver a ponerlo,cargo los parametros con las ultimas claves copiadas(a mano),seria genial 
![araergergaergaerg](https://github.com/albertobsd/keyhunt/assets/145237554/b6dd2313-a822-48d9-9e31-1f5c8eae5089)

**Comment by albertobsd** [comment]
Estoy trabajando en SaveProgress ( https://github.com/albertobsd/SaveProgress ) pero si no sabes programar no sirve que veas el codigo ahi.

**Reply by deoke1983** [reply]
por eso te lo puse en "ideas",y una pregunta que no viene al caso,hay alguna manera en "-B both " que se mantenga mas tiempo en un rango antes de saltar al otro?
me explico:
C:\Users\Propietario\Desktop\GUI_keyhunt-main\GUI_keyhunt-main>keyhunt -m bsgs -t 4 -f publickey130.txt -r 20000000000000048e85f000000000000:3ffffffffffffffce71481fffffffffff -B both -k 512 -l compress -S
[+] Version 0.2.230519 Satoshi Quest, developed by AlbertoBSD
[+] Threads : 4
[+] K factor 512
[+] Search compress only
[+] Mode BSGS both
[+] Opening file publickey130.txt
[+] Added 1 points from file
[+] Range
[+] -- from : 0x20000000000000048e85f000000000000
[+] -- to   : 0x3ffffffffffffffce71481fffffffffff
[+] N = 0x100000000000
[+] Bloom filter for 2147483648 elements : 7361.33 MB
[+] Bloom filter for 67108864 elements : 230.04 MB
[+] Bloom filter for 2097152 elements : 7.19 MB
[+] Allocating 32.00 MB for 2097152 bP Points
[+] Reading bloom filter from file keyhunt_bsgs_4_2147483648.blm .... Done!
[+] Reading bloom filter from file keyhunt_bsgs_6_67108864.blm .... Done!
[+] Reading bP Table from file keyhunt_bsgs_2_2097152.tbl .... Done!
[+] Reading bloom filter from file keyhunt_bsgs_7_2097152.blm .... Done!
[+] Thread 0x3ffffffffffffffcd05185fffffffffff   nds: ~70 Pkeys/s (70486073287854105 keys/s)

el programa no se si cambia cada X llaves....o cada X tiempo,¿hay algun modo de decirle....cambia cada 10 segundos...o cada 500 pkeys?

aunque no se programar,le echare un ojo aunque solo sea para marearme a tu "save progress"^^

**Comment by albertobsd** [comment]
`both` es por forward and backward en forma sequential, esto es que solo se mueve hasta el siguiente rango una vez que terminó con el actual

**Reply by deoke1983** [reply]
entiendo,entonces no se le puede decir.....buscame hacia adelante....5 rangos....y hacia atras....25 rangos.ok gracias!
te mandé un correo,echale un ojo un saludo!
